.github/workflows/CI.yml

```diff
@@ -13,10 +13,13 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.1'
-#          - 'nightly'
+          - '1.6'
+          - '1'
+          - 'nightly'
```

Comment:
I don't like testing against `nightly`, see https://discourse.julialang.org/t/why-do-packages-run-continuous-integration-tests-on-julia-nightly/101208/3?u=tim.holy. I know this isn't in General but it probably will be soon.